20_1 DataGrid: enterKeyAction is also available for for the "row" and "form" edit modes (#1927)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/1 Configuration/keyboardNavigation/enterKeyAction.md

---
---
##### shortDescription

> Specifies whether the Enter key switches the cell or row to the editing state or moves focus in the [enterKeyDirection](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/keyboardNavigation/enterKeyDirection.md '{basewidgetpath}/Configuration/keyboardNavigation/#enterKeyDirection'). Applies if **editing**.[mode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/mode.md '{basewidgetpath}/Configuration/editing/#mode') is *"cell"*, *"batch"*, *"row"*, or *"form"*.

Reviewer: ```suggestion
Specifies whether the Enter key switches the cell or row to the editing state or moves focus in the [enterKeyDirection](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/keyboardNavigation/enterKeyDirection.md '{basewidgetpath}/Configuration/keyboardNavigation/#enterKeyDirection'). Applies for all [edit modes](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/mode.md '{basewidgetpath}/Configuration/editing/#mode'), except *"popup"*.
```

---
---
##### shortDescription

> Specifies whether the Enter key switches the cell or row to the editing state or moves focus in the [enterKeyDirection](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/keyboardNavigation/enterKeyDirection.md '{basewidgetpath}/Configuration/keyboardNavigation/#enterKeyDirection'). Applies for all [edit modes](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/mode.md '{basewidgetpath}/Configuration/editing/#mode'), except *"popup"*.

Reviewer 2: ```suggestion
Specifies whether the Enter key switches the cell or row to the edit state or moves focus in the [enterKeyDirection](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/keyboardNavigation/enterKeyDirection.md '{basewidgetpath}/Configuration/keyboardNavigation/#enterKeyDirection'). Applies for all [edit modes](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/mode.md '{basewidgetpath}/Configuration/editing/#mode'), except *"popup"*.
```
